js/exporter/exceljs/export_pivot_grid.js

```diff
@@ -1,47 +1,238 @@
-import { isDefined, isObject } from '../../core/utils/type';
+import { isDefined, isObject, isFunction } from '../../core/utils/type';
 import { Export } from './export';
-import { noop } from '../../core/utils/common';
+import { getDefaultAlignment } from '../../core/utils/position';
+import { camelize } from '../../core/utils/inflector';
 
-const helpers = {
-    _getWorksheetFrozenState(dataProvider, cellRange) {
-        return { state: 'frozen', xSplit: cellRange.from.column + dataProvider.getFrozenArea().x - 1, ySplit: cellRange.from.row + dataProvider.getFrozenArea().y - 1 };
-    },
+const FIELD_HEADERS_SEPARATOR = ', ';
+
+class PivotGridHelpers {
+    constructor(worksheet, dataProvider, options) {
+        this.worksheet = worksheet;
+        this.dataProvider = dataProvider;
+
+        this.topLeftCell = options.topLeftCell;
+
+        this.mergeColumnFieldValues = options.mergeColumnFieldValues;
+        this.mergeRowFieldValues = options.mergeRowFieldValues;
+
+        this.exportFilterFieldHeaders = options.exportFilterFieldHeaders;
+        this.exportDataFieldHeaders = options.exportDataFieldHeaders;
+        this.exportColumnFieldHeaders = options.exportColumnFieldHeaders;
+        this.exportRowFieldHeaders = options.exportRowFieldHeaders;
```

Comment:
hm, didn't find using of this properties. Are they useful?

Reply:
They uses inside _tryGetFieldHeaders() method